### PR TITLE
Updates - added note for new repos

### PR DESCRIPTION
**Pull request by thesurenk**
Added these lines:

<Aside type="note">
If you have created a new repository, make sure you have created and pushed the `main` or `master` branch to it. The branch can be empty.
</Aside>

If user is using a new repository without any branches, user will not be able to proceed as "Production branch" dropdown would be empty.

If a new repository, without any branches:
Screen_Shot_2022-02-03_at_11_20_20_AM

After adding a "main" branch:
Screen_Shot_2022-02-03_at_11_25_49_AM

### CODE REVIEW

File: products/pages/src/content/get-started/index.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Your **project name** will be used to generate your project's hostname. By defau
 
 **Production branch** indicates the branch that Cloudflare Pages should use to deploy the production version of your site. For most projects, this is the `main` or `master` branch. All other branches that are not your production branch will be used for [preview deployments](/platform/preview-deployments).
 
+<Aside type="note">
+If you have created a new repository, make sure you have created and pushed the `main` or `master` branch to it. The branch can be empty.
```
Comment (lukeed) [inline products/pages/src/content/get-started/index.md]:
```suggestion

You must have pushed at least one branch to your GitHub or GitLab project in order to select a **Production branch** from the dropdown menu.
```

Comment (deadlypants1973) [reply products/pages/src/content/get-started/index.md]: Changed production to Production to reflect UI.

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]
Requesting content update.

**Review by lukeed — APPROVED** [review]

File: products/pages/src/content/get-started/index.md [hunk 2]
```diff
@@ -53,6 +53,13 @@ Your **project name** will be used to generate your project's hostname. By defau
 
 **Production branch** indicates the branch that Cloudflare Pages should use to deploy the production version of your site. For most projects, this is the `main` or `master` branch. All other branches that are not your production branch will be used for [preview deployments](/platform/preview-deployments).
 
+<Aside type="note">
+
+You must have pushed at least one branch to your GitHub or GitLab project in order to select a **Production branch** from the dropdown menu.
+
+</Aside>
+
+
```
Comment (deadlypants1973) [inline products/pages/src/content/get-started/index.md]:
```suggestion
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**